docs/sidebars.js

```diff
@@ -195,6 +195,7 @@ const sidebars = {
         "dev_docs/getting_started/noir",
         "dev_docs/getting_started/sandbox",
         "dev_docs/getting_started/cli",
+        "dev_docs/getting_started/noir_contracts",
```

Comment:
Ordering wise, you could probably move this up above the cli. Logically, would like it to be right after noir, but since you are showing things into sandbox directories probably fine to have it there. 

This is more a comment/reflection that something that necessarily needs changing.